rllib/examples/env/bandit_envs_recommender_system.py

```diff
@@ -127,9 +127,9 @@ def step(self, action):
         # Action is the suggested slate (indices of the docs in the
         # suggested ones).
 
-        scores = [
-            np.dot(self.current_user, doc) for doc in self.currently_suggested_docs
-        ]
+        scores = softmax(
```

Comment:
The scores can be > 1. If we then select a score to calculate the regret and reward, the reward may become < 0.

Reply:
Sorry for not being more verbose about this.

Reply:
The way we calculate the reward here makes it so that the reward ends up somewhere <= 1, instead of between 0 and 100 like it is specified in the observation space above.